components/Base/FileUpload.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div>
+    <label
+      class="flex flex-col items-center justify-center w-full bg-gray-100 border-[2px] border-dashed h-20 rounded-3xl"
+      :accept="accept"
+      draggable="true"
+      @dragstart="handleDragStart"
+      @dragenter="handleDragEnter"
+      @dragleave="handleDragLeave"
+      @dragover="handleDragOver"
+      @drop.prevent="handleDrop"
+      @change="handleChange"
+    >
+      <slot v-if="!selectedFile" />
+      <p
+        v-else
+        class="text-xs"
+      >
+        {{ selectedFile.name }}
+      </p>
```

Comment:
Here, as you see I can add `.torrent` file, but I shouldn't. 

Also add padding :)

![image](https://user-images.githubusercontent.com/103642381/224854452-1e91d2f2-0018-44bf-8a05-b38fc102dbe5.png)